lms/templates/hidden_content.html

```diff
@@ -6,14 +6,14 @@
 
 <div class="sequence hidden-content proctored-exam completed">
   <h3>
-      ${_("The due date for this {subsection_format} has passed.").format(
+      ${_("The date for this {subsection_format} has passed.").format(
```

Comment:
Saying "date has passed" without calling it a "due date" seems confusing.

Is it possible here to address both cases separately? (At both line 9 and line 16)
"The due date ... has passed" and "The course has ended"?